PSET-3/Sreekanth-Solution/Sreekanth_PSET3_solution.py

```diff
@@ -0,0 +1,41 @@
+<<<<<<< HEAD
```

Comment:
Please don't import any package or module

Reply:
Remove HEAD

Reply:
Removed all the import function and solved the answer without using .split and .partition methods.